libraries/namespaces.raml

```diff
@@ -0,0 +1,7 @@
+#%RAML 1.0 Library
+annotationTypes:
+  namespace:
+    description: |
+      this annotation type allows you to associate unique identifier with a RAML document.
+    type: string
+(namespace): http://raml.org/vocabularies/annotations/core
```

Comment:
>Not really sure what you mean. Can you elaborate, please?

The global goal of this annotation is to solve **identity** problem for reusable artefacts in RAML. Solving this goal does not strictly require us to add any restrictions on the identifier 
values , we can require it to be an `uri` or `url` or just leave it as string. By some reason I have misread you and thought that you are suggesting it to be `urls` which looks like to restrictive at this stage.

My suggestion was to let it be a string with a comment in the initial version, and then when we gather more feedback, release a next version which will contain stricter restriction.

Only concern that I have agains making it an `uri` in a first version it that it might affect some theoretical naming schemes that people may want to use.

Regards,
Pavel